core/vm/privacy/bulletproof.go

```diff
@@ -1385,6 +1453,11 @@ func genECPrimeGroupKey(n int) CryptoParams {
 		cg}
 }
 
+func (Point *ECPoint) isZeroPoint() bool {
+	zero := big.NewInt(0)
+	return Point.X.Cmp(zero) == 0 && Point.Y.Cmp(zero) == 0
```

Comment:
We can use `common.Big0` instead of `zero := big.NewInt(0)`:

```go
    return Point.X.Cmp(zero) == common.Big0 && Point.Y.Cmp(zero) == common.Big0
```